Unable to process 'Scene' NSP releases

**[ERROR]      at Switch_Backup_Manager.Util.GetFileDataNSP(String file)**

Tested not working with:
`0003 - Sonic Mania (World) (En,Ja,Fr,De,Es,It) (eShop).nsp`
`0004 - Resident Evil Revelations 2 (World) (En,Ja,Fr,De,Es,It,Pt,Zh,Ko,Ru) (eShop).nsp`
`0054 - Unepic (World) (En,Fr,De,Es,It,Pt,Ru) (eShop).nsp`
`0102 - Bomber Crew (USA, Europe) (En,Fr,De,Es,Ru) (eShop).nsp`
Plus ALL **ACA NEOGEO**, and ALL **Arcade Archives** releases.

All acquired via Telegram.

Also tested with XCI Explorer 1.4.0 and 1.4.1 - also not working.

**Logs:**

`2018-08-25 16:39:20.726 [INFO]    Adding 1 files on local Eshop database.`
`2018-08-25 16:39:20.740 [INFO]    Adding NSP file: 0102 - Bomber Crew (USA, Europe) (En,Fr,De,Es,Ru) (eShop)`
`2018-08-25 16:39:20.750 [DEBUG]   Analyzing xml file.`
`2018-08-25 16:39:20.757 [DEBUG]   Analyzing xml file.`
`2018-08-25 16:39:20.765 [ERROR]      at Switch_Backup_Manager.Util.GetFileDataNSP(String file)`
`2018-08-25 16:39:20.771 [INFO]    Finished adding files. Total time was 00:00.03.`
`2018-08-25 16:39:20.779 [DEBUG]   searching for 01007900080B6000 on database.`
`2018-08-25 16:39:20.785 [INFO]    01007900080B6000 was already on database. Ignoring.`

`2018-08-25 16:39:39.166 [INFO]    Adding 1 files on local Eshop database.`
`2018-08-25 16:39:39.173 [INFO]    Adding NSP file: 0004 - Resident Evil Revelations 2 (World) (En,Ja,Fr,De,Es,It,Pt,Zh,Ko,Ru) (eShop)`
`2018-08-25 16:39:39.181 [DEBUG]   Analyzing xml file.`
`2018-08-25 16:39:39.187 [DEBUG]   Analyzing xml file.`
`2018-08-25 16:39:39.194 [ERROR]      at Switch_Backup_Manager.Util.GetFileDataNSP(String file)`
`2018-08-25 16:39:39.202 [INFO]    Finished adding files. Total time was 00:00.02.`
`2018-08-25 16:39:39.208 [DEBUG]   searching for 010095300212A000 on database.`
`2018-08-25 16:39:39.214 [DEBUG]   010095300212A000 not found on database. Adding...`
`2018-08-25 16:39:39.221 [DEBUG]   Adding element...`
`2018-08-25 16:39:39.226 [DEBUG]   Saving xml SBM_NSP_Local.xml`
`2018-08-25 16:39:39.244 [DEBUG]   xml saved...`


![sbm scene releases 01](https://user-images.githubusercontent.com/522690/44617044-41556680-a886-11e8-864a-fe12345c23b2.png)
![sbm scene releases 02](https://user-images.githubusercontent.com/522690/44617147-f4728f80-a887-11e8-871f-3aa0ffed7b8b.png)

--- a/Switch Backup Manager/Util.cs
+++ b/Switch Backup Manager/Util.cs
@@ -1790,7 +1790,7 @@ namespace Switch_Backup_Manager
                     array3[n].Name = new string(chars.ToArray());
                     chars.Clear();
 
-                    if (array3[n].Name.EndsWith(".xml"))
+                    if (array3[n].Name.EndsWith(".cnmt.xml"))
                     {
                         logger.Debug("Analyzing xml file.");
                         byte[] array4 = new byte[array3[n].Size];

